db/migrate/20191213094311_add_document_ref_to_whitehall_imports.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class AddDocumentRefToWhitehallImports < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :whitehall_imports,
+                  :document,
+                  foreign_key: { on_delete: :cascade },
```

Comment:
It'd be more consistent with https://github.com/alphagov/content-publisher/blob/master/docs/adr/0013-permitted-deletion.md to use restrict since this feature isn't related to access limited. That builds off https://github.com/alphagov/content-publisher/blob/master/docs/adr/0008-restricted-deletion.md where restrict was just set as the default everywhere so we don't get too bogged down in whether things should be able to delete others.

A later concern that we should log though is that we may need to delete WhitehallImports that pull in access limited data as part of the data sync (what an annoying complication)